documentation/guide-kafka.asciidoc

```diff
@@ -6,393 +6,7 @@ toc::[]
 Messaging Services provide an asynchronous communication mechanism between applications. Technically this is
 implemented using http://kafka.apache.org/documentation.html/[Apache Kafka] .
 
-Devonfw uses link:https://spring.io/projects/spring-kafka[Spring-Kafka] as kafka framework.
+For link:spring.asciidoc[spring], devonfw uses link:https://spring.io/projects/spring-kafka[Spring-Kafka] as kafka framework.
+For more details, check the link:spring/guide-kafka-spring.asciidoc[devon4j-kafka].
 
-This guide explains how Spring kafka is used in devonfw applications. It focuses on aspects which are special to devonfw if you want to learn about spring-kafka you should adhere to springs references documentation.
```

Comment:
Maybe we can add some general words about Kafka in this guide. Also the parts about naming conventions or parallel processing are not really spring or devon4j specific.

Reply:
I see but I find it difficult for user to follow if we just seperate those aspects.

Reply:
Kafka